src/task/ReceiveVideoInputTask.ts

```diff
@@ -32,7 +32,6 @@ export default class ReceiveVideoInputTask extends BaseTask {
     if (!this.context.videoTileController.hasStartedLocalVideoTile()) {
       this.context.logger.info('has not started local video tile');
       if (this.context.activeVideoInput) {
-        this.stopVideoInput();
```

Comment:
At this line, we know that an attendee has not enabled a video because the `hasStartedLocalVideoTile` returns `false`. That's why you are removing this line. Do I understand correctly?

Reply:
Updated the code to remove the activeVideoInput line but mainly we no longer try to release the video stream anymore.